packages/core/src/builtin-tasks/node.ts

```diff
@@ -31,9 +39,31 @@ subtask('node:start', 'start a ckb node')
       genesisAccountArgs: genesisArgs,
     })
 
-    env.config.network = {
-      url: 'http://localhost:' + port,
+    env.config.network = { url: ckbDockerNetwork.url }
+
+    const builtInDirPath = cachePath('built-in')
+    for (const script of BUILTIN_SCRIPTS) {
+      const filePath = path.join(builtInDirPath, script)
+      if (!fs.existsSync(filePath)) {
+        download(
+          `https://github.com/Daryl-L/ckb-production-scripts/releases/download/scripts/${script}`,
+          builtInDirPath,
+          { filename: script },
+        )
+      }
     }
+    await new Promise((resolve) => setTimeout(resolve, 20000))
+    await ckbDockerNetwork.generateLumosConfig()
+    config.initializeConfig(ckbDockerNetwork.lumosConfig)
+
+    await ckbDockerNetwork.deployScripts({
+      builtInScriptName: BUILTIN_SCRIPTS,
+      configFilePath: filePath ?? path.resolve(configPath(), 'scripts.json'),
+      builtInDirPath,
+      indexer: new Indexer(`${ckbDockerNetwork.url}/indexer`),
+      rpc: new RPC(`${ckbDockerNetwork.url}/rpc`),
+      privateKey: '0xfd686a48908e8caf97723578bf85f746e1e1d8956cb132f6a2e92e7234a2a245',
```

Comment:
How is the private key from

Reply:
> How is the private key from

It is from the file `dev-template.toml`

Reply:
It would be good to have a reference to the origin so this info won't be lost